src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -190,6 +190,20 @@
     </BoolProperty.Metadata>
   </BoolProperty>
 
+  <BoolProperty Name="AutoGenerateBindingRedirects"
+                DisplayName="Auto-generate binding redirects"
+                Description="Enables binding redirection to be added automatically to the app configuration file to override assembly unification."
```

Comment:
```suggestion
                DisplayName="Auto-generate binding redirects"
                Description="Add binding redirects automatically to App.config."
```

For `BoolProperty` the `Description` is used as the label of the check box. In general check boxes have a "directive" statement. There are a few violations of this currently which I will propose fixes for separately.